.buildkite/llvm_passes.yml

```diff
@@ -0,0 +1,34 @@
+# These steps should only run on `sandbox.jl` machines, not `docker`-isolated ones
+# since we need nestable sandboxing.  The rootfs images being used here are built from
+# the `.buildkite/rootfs_images/llvm-passes.jl` file.
+agents:
+  queue: "julia"
+  # Only run on `sandbox.jl` machines (not `docker`-isolated ones) since we need nestable sandboxing
+  sandbox.jl: "true"
+  os: "linux"
+
+steps:
+  - label: "analyzegc"
+    plugins:
+      - staticfloat/sandbox#v1:
+          rootfs_url: https://github.com/JuliaCI/rootfs-images/releases/download/v1/llvm-passes.tar.gz
+          rootfs_treehash: "2b03c3cfda5eec0c379d63b59fd84b6a8fcbecf8"
+    commands: |
+      echo "--- Install in-tree LLVM dependencies"
+      make -j 6 -C deps install-llvm install-clang install-llvm-tools install-libuv install-utf8proc install-unwind
+      echo "+++ run clangsa/analyzegc"
+      make -j 6 -C test/clangsa
+      make -j 6 -C src analyzegc
+    timeout_in_minutes: 60
+
+  - label: "llvmpasses"
+    plugins:
+      - staticfloat/sandbox#v1:
+          rootfs_url: https://github.com/JuliaCI/rootfs-images/releases/download/v1/llvm-passes.tar.gz
```

Comment:
Does this go through our CDN? Can we cache these?

Reply:
Yes, it should automatically get cached.  It gets downloaded as an artifact, and all artifacts are automatically cached as part of the `JuliaCI/julia` plugin.